src/rpc/http2.md

```diff
@@ -0,0 +1,56 @@
+# HTTP/2
+
+HTTP/2 evolved out of Google's experimental SPDY protocol as a successor to HTTP/1.1. Since it maintains high-level compatability with HTTP/1.1 let's first take a look at HTTP/1.x.

```

Comment:
I would start with a high-level summary of what HTTP 2 is and how it works, rather than starting with the history and then background